File: Tundra/DoubleBufferedFlowLayoutPanel.vb
Imports System.Windows.Forms

Public Class DoubleBufferedFlowLayoutPanel
    Inherits FlowLayoutPanel
    Public Sub New()
        DoubleBuffered = True
        SetStyle(ControlStyles.SupportsTransparentBackColor, True)
        SetStyle(ControlStyles.OptimizedDoubleBuffer, True)
        SetStyle(ControlStyles.ResizeRedraw, True)
    End Sub
End Class

File: src/TundraLib/DoubleBufferedFlowLayoutPanel.cs
using System.Windows.Forms;

namespace LangPadSupport
{
    public class DoubleBufferedFlowLayoutPanel : FlowLayoutPanel
    {
        public DoubleBufferedFlowLayoutPanel()
        {
            DoubleBuffered = true;
            SetStyle(ControlStyles.SupportsTransparentBackColor, true);
            SetStyle(ControlStyles.OptimizedDoubleBuffer, true);
            SetStyle(ControlStyles.ResizeRedraw, true);
        }
    }
}